source/user-manual/api/getting-started.rst

```diff
@@ -24,54 +24,12 @@ The API starts at boot time. To control or check the **wazuh-api** service, use
-
-The interface has a list of all available endpoints separated into groups (active-response, agents, ciscat, cluster, etc.). Each endpoint can have multiple request methods (GET, POST, PUT, DELETE). To differentiate them, a specific color is used for each one, along with a label on the left. To run any endpoint, just click on it, press the *Try it out* button, fill in the necessary parameters and click on the *Execute* button. Please note that **login is required** to run endpoints. You can see how to do it in the next section, :ref:`Logging into the API <api_log_in>`.
-
-.. thumbnail:: ../../images/api/api-ui-overview.png
-    :align: center
-    :width: 100%
-
-The Wazuh API endpoints can be reached via the swagger interface or using the command-line tool *curl*. In some of the examples below we will use both options.
-
 .. _api_log_in:
 
 Logging into the API
 --------------------
 
-Wazuh API endpoints require authentication in order to be used. Therefore, all calls must include a JSON Web Token. JWT is an open standard (RFC 7519) that defines a compact and self-contained way for securely transmitting information between parties as a JSON object.
-
-Logging into the API UI
-^^^^^^^^^^^^^^^^^^^^^^^
-
-1. Access the API interface at <protocol>://<host>:<port>/ui. The default path is ``https://localhost:55000/ui``.
-2. Click on the green *Authorize* button.
-3. Enter the username and password inside the section *basicAuth  (http, Basic)*. By default, the user is ``wazuh`` and the password is ``wazuh``.
-4. Click on the green *Authorize* button under the password field and close this window.
-5. Expand the endpoint ``GET /security/user/authenticate`` and click on the *Try it out* button.
-6. Click on the *Execute* button. A result like the following will appear. Copy the value that appeared in ``<YOUR_JWT_TOKEN>``, without the quotes.
-
-    .. code-block:: json
-        :class: output
-
-        {
-          "token": "<YOUR_JWT_TOKEN>"
-        }
-
-7. Click again on the Authorize button.
-8. Paste the value of the copied token into the *jwt  (http, Bearer)* field.
-9. Click on the Authorize button inside. After that, you can use any API endpoint allowed for the roles the user belongs to.
-
-Logging in with cURL
-^^^^^^^^^^^^^^^^^^^^
+Wazuh API endpoints require authentication in order to be used. Therefore, all calls must include a JSON Web Token. JWT is an open standard (RFC 7519) that defines a compact and self-contained way for securely transmitting information between parties as a JSON object. Follow the steps below to get a token in order to run any endpoint:
 
 1. Use the cURL command to get the JWT token. Replace <user> and <password> with yours. By default, the user is ``wazuh`` and the password is ``wazuh``.  If ``SSL`` (https) is enabled in the API and it is using the default **self-signed certificates**, it will be necessary to add the parameter ``-k``.
```

Comment:
```suggestion
1. Use the cURL command to log in, the API will provide a JWT token upon sucess. Replace <user> and <password> with yours. By default, the user is ``wazuh`` and the password is ``wazuh``. If ``SSL`` (https) is enabled in the API and it is using the default **self-signed certificates**, it will be necessary to add the parameter ``-k``.
```